_draft/2017-07-31-why-jwt.md

```diff
@@ -0,0 +1,52 @@
+---
+title:  "Why JWT?"
+author: iviczl
+image: '/img/nuget-highway.jpg'
+tags: [jwt, token, authentication]
+---
+# Authentication Exposure
+
+However sensenet ECM has been supporting authentication through NTLM and Basic for a long time, neither of them is a good choice for some scenarios. NTLM can be used only, if the authenticating user is a Windows domain user as well, Basic authentication on the other hand requires to send your user's credentials with every request. 
+We decided to implement a modern and easily usable and secure enough means to authenticate a user in the sensenet ECM environment.
```

Comment:
*a mean* or *means*, without "a".
Read as: "We decided to implement a means to authenticate [...]"
or
"We decided to implement means to authenticate [...]"

Instead of "means", "ways" may be a better choice. What do you think?